apps/frontpage/app/blog/sitemap.ts

```diff
@@ -0,0 +1,19 @@
+import { MetadataRoute } from 'next';
+import Sitemapper from 'sitemapper';
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const blogXml = new Sitemapper({
+    url: 'https://storybook.js.org/blog/sitemap/sitemap-0.xml',
+    timeout: 15000, // 15 seconds
+  });
+
+  try {
+    const { sites } = await blogXml.fetch();
+    return sites.map((site) => {
+      const url = site.replace(/\/$/, '');
+      return { url };
+    });
+  } catch (error) {
```

Comment:
If there's an error we need to fail the build. We cannot ship an empty sitemap without SEO consequences.

Reply:
Applies multiple places